src/nodes.coffee

```diff
@@ -2062,6 +2062,17 @@ exports.Super = class Super extends Base
     attachCommentsToNode salvagedComments, @accessor.name if salvagedComments
     fragments
 
+  ast: (o, level) ->
+    if @accessor?
+      return (
+        new Value(
```

Comment:
For things like `super.a`, the Babel-style AST is a `MemberExpression` whose `object` is a `Super`

So for those cases, transform the `Super` with an `@accessor` to a `Value` with a property